api/middlewares/upload_middleware.go

```diff
@@ -160,24 +193,39 @@ func (u UploadMiddleware) Handle() gin.HandlerFunc {
 
 				if conf.ThumbnailEnabled {
 					thumbReader := bytes.NewReader(fileByte)
+					errGroup, ctx := errgroup.WithContext(c.Request.Context())
+
```

Comment:
@Denes-cilwal Why is another `errgroup` created can't it be done with `errGroup` that was defined previously? 🤔

Reply:
> @Denes-cilwal Why is another `errgroup` created can't it be done with `errGroup` that was defined previously? 🤔

yes no need @dipeshdulal  sir . Thank you for correction!!